src/calculateRentalCost.js

```diff
@@ -5,6 +5,24 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const perDay = 40;
```

Comment:
The variable `perDay` should be declared as a constant outside of the function to avoid redeclaration every time the function is called and to indicate that it is a constant value. Consider renaming it to `DAILY_RENTAL_RATE` for better clarity.

Reply:
I agree with this suggestion made by AI